README.md

```diff
@@ -44,4 +44,5 @@ To get a test coverage report:
 
 ## Todo
 
-* Make app names lowercase to conform to Python style guide ([PEP8](https://www.python.org/dev/peps/pep-0008/)).
\ No newline at end of file
+* Make app names lowercase to conform to Python style guide ([PEP8](https://www.python.org/dev/peps/pep-0008/)).
+* Fix add external user
```

Comment:
This is already written in the code as a `todo` line.

Reply:
True.